### ISSUE TITLE
Export notes as a doc 

### ISSUE THREAD

**Issue by jonasbrant** [Meeds-io/meeds#477]
The notes (former wiki) is a great feature for collaboration in shared public information. 
but during the migration from wiki to note  we lost some material. I believe that include the possibility to export notes as word document or other type could be useful in the day by day activity to share this materials

**Comment by srenault-meeds** [comment]
Hi @jonasbrant 
Thanks for reporting this issue.
Actually, you can currently export any notes to pdf. To do such action, please follow these steps:
- Access a note
- From the three dots (top-right of the note page), you should have an option called "Print to PDF"

Can you please tell us if that fits your request?
Thanks by advance

**Comment by jonasbrant** [comment]
this is great

**Comment by jonasbrant** [comment]
I believe be able to export in a editable form is an advantage but maybe no a priority at this moment

**Comment by srenault-meeds** [comment]
Yes sure, can be studied indeed. We might think as well about a way to invite a guest to access the note without giving him access to the whole platform.
(By the way, in case you don't know, you can invite external users if needed to a space so they can collaborate with you in a secure workspace without having access to the whole platform).
